Sources/FluentUI_iOS/Components/BottomSheet/BottomSheetController.swift

```diff
@@ -696,6 +719,14 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
             stackView.bottomAnchor.constraint(equalTo: bottomSheetContentView.bottomAnchor)
         ])
 
+        if let headerContentView {
+            resizingHandleContentOverlapConstraints = headerContentView.topAnchor.constraint(equalTo: resizingHandleView.bottomAnchor, constant: -currentResizingHandleHeight)
+        } else {
+			resizingHandleContentOverlapConstraints = expandedContentView.topAnchor.constraint(equalTo: resizingHandleView.bottomAnchor, constant: -currentResizingHandleHeight)
```

Comment:
we'll need to account for new height in expandedSheetHeight/collapsedSheetHeight